src/resources/startup.py

```diff
@@ -40,6 +40,10 @@
 from grpc_channelz.v1 import channelz_pb2
 from grpc_channelz.v1 import channelz_pb2_grpc
 
+from grpc_channelz.v1 import channelz
```

Comment:
Why do you import these again?

Reply:
I think it. I think I made a mistake when merging the conflicts.